internal/observability/event/eventer.go

```diff
@@ -259,11 +261,38 @@ func NewEventer(log hclog.Logger, serializationLock *sync.Mutex, serverName stri
 			}
 		}
 		if addToAudit {
+			var fop AuditFilterOperations
+			if s.AuditConfig != nil {
+				fop = s.AuditConfig.FilterOverrides
+			}
+			s.AuditConfig, err = NewAuditConfig(WithAuditWrapper(opts.withAuditWrapper), WithFilterOperations(fop))
+			if err != nil {
+				return nil, fmt.Errorf("%s: %w", op, err)
+			}
+			encryptFilter := &encrypt.Filter{
+				Wrapper: opts.withAuditWrapper,
+			}
+			if len(s.AuditConfig.FilterOverrides) > 0 {
+				overrides := encrypt.DefaultFilterOperations()
+				for k, v := range s.AuditConfig.FilterOverrides {
+					overrides[encrypt.DataClassification(k)] = encrypt.FilterOperation(v)
+				}
+				encryptFilter.FilterOperationOverrides = overrides
+			}
+			id, err := NewId("encrypt-audit")
```

Comment:
Hum... this is a really excellent question.  It hadn't occured to me that operators may want and non-redacted audit sink.  I just checked the go-eventlogger library and currently there's no way to set a filter to "no operation", even with an explicit override that specifies "no operation".  That's a defect and I'll get a PR open to fix that.   I also think, the node in go-eventlogger should check if all the filters are set to "no operation" and short-circuit the node to immediately return the unfiltered event.   If we go that route, then we don't have to worry about checking/changing things here.  Thanks again for the insightful question!

Reply:
I actually wasn't thinking "not redacted" so much as "not encrypted" -- e.g. hmac'd or something. It just seemed based on this that if we were doing something other than encrypting it would still be `encrypt-audit`...?

Reply:
I think so... it's an `encrypt.Filter` node for auditing.